Translations/glossary.md

```diff
@@ -0,0 +1,73 @@
+# Translations glossary
+
+## Variables
+
+There are two types of variables in the text files: %s, %i. All of these will add something within the single string, here are some examples. 
+
+`%s` will always be another word. Example: `%s of %s` refers to an created Item which includes affixes. `Tin Cap of the Fool` would be such an item.
```

Comment:
A created item ;(

Reply:
For words with vowels (and sometimes "y") should "an".  This rule is generally a bit weak (e.g. if a word contains silent letters such as "hour" it will omit the consonant "h" and become "an hour") as well as the other way around for some words (e.g. weak vowel followed by a pronounced consonant such as in the word "one" - "a one-room appartment" but "an item" is correct here.

Other examples:

"I heard an item on the radio about women engineers."
"[to be] an item" - idiom suggesting a romantic involvement

:D